packages/components/src/components/contract-card/contract-card-items/toggle-card-dialog.tsx

```diff
@@ -100,30 +108,43 @@ const ToggleCardDialog = ({
         <Icon
             className='dc-contract-card-dialog-toggle__icon'
             icon='IcEdit'
-            color={is_valid_to_cancel ? 'disabled' : ''}
+            color={is_adding_disabled ? 'disabled' : ''}
             size={12}
         />
     );
 
     return (
         <div onClick={handleClick}>
-            {is_valid_to_cancel && should_show_cancellation_warning && isDesktop() ? (
+            {is_risk_management_disabled && isDesktop() ? (
                 <Popover
                     alignment='right'
                     classNameBubble='dc-contract-card-dialog__popover-bubble'
-                    className='dc-contract-card-dialog__popover'
+                    className={classNames('dc-contract-card-dialog__popover', {
+                        'dc-contract-card-dialog__popover--accumulator': is_accumulator,
+                    })}
                     is_bubble_hover_enabled
                     margin={2}
                     zIndex='2'
-                    message={getCardLabels().TAKE_PROFIT_LOSS_NOT_AVAILABLE}
+                    message={notificationText}
                     onBubbleClose={onPopoverClose}
                 >
-                    <button ref={toggle_ref} className='dc-contract-card-dialog-toggle' onClick={toggleDialogWrapper}>
+                    <button
+                        ref={toggle_ref}
+                        className='dc-contract-card-dialog-toggle'
+                        onClick={toggleDialogWrapper}
+                        disabled={is_risk_management_disabled}
+                    >
                         {edit_icon}
                     </button>
                 </Popover>
             ) : (
-                <button ref={toggle_ref} className='dc-contract-card-dialog-toggle' onClick={toggleDialogWrapper}>
+                <button
+                    ref={toggle_ref}
+                    className={classNames('dc-contract-card-dialog-toggle', {
+                        'dc-contract-card-dialog-toggle--disabled': is_risk_management_disabled,
```

Comment:
We are disabling button here with the usage of css (not with the attr `disabled`), because we need to give an opportunity to click on this button on mobile in order to show notification with text explanation